Okta.Core.Tests/Clients/Helpers.cs

```diff
@@ -155,5 +156,36 @@ internal static Models.Group CreateGroup(OktaClient oktaClient, TestGroup dbGrou
             return oktaGroup;
         }
 
+        internal static string GetRandomString()
+        {
+            const string AllowedChars = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz#@$^*()";
+            Random rng = new Random();
+
+            var randomStrings = RandomStrings(AllowedChars, 8, 16, 1, rng);
+
+            return randomStrings.ElementAt<string>(0);

```

Comment:
If you just want a single random string here, why not just make RandomStrings return a single string instead of an Enumerator with multiple?